src/_articles/archive/zones.md

```diff
@@ -165,16 +171,24 @@ Another way to produce an uncaught error is
 to call `new Future.error()` or
 a Completer's `completeError()` method.
 
-Use the `onError` argument to `runZoned()` to
-install a _zoned error handler_—an asynchronous error handler
+`runZoned()` does not handle errors.
+Its error-handling parameter `onError` 
+was deprecated because it requires the
+function's return type to always be nullable,
+which is unneccesarily cumbersome for [Null Safe code][]. 
```

Comment:
**question:** When was `onError` deprecated and by whom? If this deprecation occurred recently, call that out.

**suggestion:** Provide context and tighten up the writing.

```suggestion
`runZoned()` deprecated its error-handling parameter `onError`.
That parameter required the function's return type to be nullable.
That requirement proves cumbersome for [Null Safe code][]. 
```

Reply:
It happened as part of Dart 2.12 (null safety), because the existing API just didn't work very well with null safety.

Reply:
going to rewrite this whole section per Lasse's comment below